recipes-domd/agl/files/meta-agl-bsp/conf/include/agl_rcar_xt.inc

```diff
@@ -28,7 +28,7 @@ PREFERRED_VERSION_gstreamer1.0-plugins-good = "1.6.3"
 PREFERRED_VERSION_gstreamer1.0-plugins-ugly = "1.6.3"
 PREFERRED_VERSION_gstreamer1.0-plugins-vspfilter = "1.0%"
 
-MACHINE_FEATURES_append = " multimedia"
+#MACHINE_FEATURES_append = " multimedia"
```

Comment:
Is this a part of adding vGPU support to AGL?

Reply:
will fix